app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,9 @@
 <resources>
     <string name="app_name">Github Subscribers</string>
     <string name="splash_title">GITHUB SUBSCRIBERS</string>
-    <string name="api_client_error">Connection problems</string>
+    <string name="api_client_error_no_connection">Connection problems</string>
```

Comment:
If these are the error message the user will see, please be user-friendly, let's say, `We are having connections issues, please try again later.`